test/unittest/unittest_nntrainer_lr_scheduler.cpp

```diff
@@ -126,7 +146,27 @@ TEST(lr_exponential, prop_01_n) {
  * @brief test set and get learning rate
  *
  */
-TEST(lr_exponential, prop_02_p) {
+TEST(lr_exponential, prop_02_n) {
```

Comment:
Can you delete this test cause I made same one

Reply:
okay